server/sonar-web/src/main/js/components/common/BranchStatus.tsx

```diff
@@ -77,3 +96,15 @@ export default function BranchStatus({ branch, concise = false }: Props) {
     return <Level level={branch.status.qualityGateStatus} small={true} />;
   }
 }
+
+function getQualityGateColor(status: string) {
+  let indicatorColor = 'gray';
```

Comment:
when is it gray?

Reply:
When the quality gate status is NONE (if it ever can be). I just based myself on https://github.com/SonarSource/sonarqube/blob/b3cadbeb532ecb3828a68b8d4d8eb2bcff2f8d9f/server/sonar-web/src/main/js/components/ui/Level.css#L76